packages/build-utils/src/prerender.ts

```diff
@@ -165,5 +168,15 @@ export class Prerender {
       }
       this.experimentalStreamingLambdaPath = experimentalStreamingLambdaPath;
     }
+
+    if (experimentalFallbackMetadataPrefixLength !== undefined) {
+      if (typeof experimentalFallbackMetadataPrefixLength !== 'number') {
+        throw new Error(
+          'The `experimentalFallbackMetadataPrefixLength` argument for `Prerender` must be a string.'
```

Comment:
The error says the value must be a string, but the conditional above says it must be a number.

```suggestion
          'The `experimentalFallbackMetadataPrefixLength` argument for `Prerender` must be a number.'
```